python/paddle/fluid/tests/unittests/mkldnn/test_pool2d_mkldnn_op.py

```diff
@@ -61,6 +61,22 @@ def init_data_type(self):
 create_test_mkldnn_class(TestCase5)
 
 
+class TestAvgPoolAdaptive(TestPool2D_Op):
+    def init_adaptive(self):
+        self.adaptive = True
+
+    def init_pool_type(self):
+        self.pool_type = "avg"
+        self.pool2D_forward_naive = avg_pool2D_forward_naive
+
+    def init_kernel_type(self):
+        self.use_mkldnn = True
+
+    def init_test_case(self):
+        self.ksize = [1, 1]
+        self.strides = [1, 1]
```

Comment:
I'm wondering if you could put test case triggering another calculation in additon to happy path in the mkldnn_reuse when adaptive is used.

Reply:
@arlesniak I have added another test. Thanks for pointing out missing scenario coverage